### PR TITLE
improve sendcommand

### PR DESCRIPTION
**Pull request by Syrooo**
Hello, I have an infrared tuya device. I tryied to control my TV with tinytuya, it did not work (smart life app is working ofc). So I look into https://iot.tuya.com/cloud/ > my app > devices > debug device > device debugging and look the API commands send by the javascript .

All of that for this little change with URI=iot-03/devices/DEVID/commands if does not work, with URI=devices/DEVID/commands it works.

I hope it will help some other persons.

### CODE REVIEW

**Comment by Syrooo** [comment]
AN exemple of my local test:

`from tinytuya import *

# Connect to Tuya Cloud
c = Cloud(
        apiRegion="eu", 
        apiKey="KEY", 
        apiSecret="SECRET")

# Select a Device ID to Test
id = "DEVID" #tv


# Send Command - Turn on switch
commands = {
	'commands': [
  {
    "code": "Volume-"
  }
]
}

print("Sending command...")
result = c.sendcommand(id,commands,uri='devices/')
print("Results\n:", result)`

File: tinytuya/Cloud.py [hunk 1]
```diff
@@ -481,7 +481,7 @@ def getdps(self, deviceid=None):
             )
         return response_dict
 
-    def sendcommand(self, deviceid=None, commands=None):
+    def sendcommand(self, deviceid=None, commands=None, uri='iot-03/devices/'):
```
Comment (jasonacox) [inline tinytuya/Cloud.py:484]: Looks clean and a fairly safe way to added additional customization (uri).  

@uzlonewolf any concerns?

**Comment by jasonacox** [comment]
Thanks for this @Syrooo !  Nice addition.

**Comment by jasonacox** [comment]
This change included in v1.12.1 (see https://pypi.org/project/tinytuya/1.12.1/).

**Pull request merged**